docs/index.html

```diff
@@ -564,5 +564,17 @@ <h2>Internationalization Considerations</h2>
         take into account when utilizing this data model.
       </p>
     </section>
+
+    <section class="appendix">
+      <h2>Acknowledgements</h2>
+      <p>
+        Portions of the work on this specification have been funded by the
+        United States Department of Homeland Security's (US DHS) Silicon Valley
+        Innovation Program under contracts 70RSAT20T00000044, 70RSAT20T00000003, and
+        70RSAT20T00000033. The content of this specification does not
```

Comment:
```suggestion
        Innovation Program under contracts
        70RSAT20T00000003,
        70RSAT20T00000033,
        and
        70RSAT20T00000044.
        The content of this specification does not
```
Keeping the contracts in alphanumerical order and stacked as above in the HTML source makes it easier to check whether one is already listed, avoiding duplication, and speeding edits when needed.